cdc/Makefile

```diff
@@ -1,7 +1,5 @@
 ### Makefile for ticdc
-.PHONY: build test check clean fmt cdc kafka_consumer coverage \
-	integration_test_build integration_test integration_test_mysql integration_test_kafka bank \
-	dm dm-master dm-worker dmctl dm-portal dm-syncer dm_coverage
+.PHONY: build uint_test lint
```

Comment:
There should be `test check clean fmt cdc` here.

Reply:
Done